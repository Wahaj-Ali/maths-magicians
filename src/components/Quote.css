.quotePage {
  display: flex;
  flex-direction: column;
  padding: 40px;
  justify-content: center;
  align-items: center;
  margin: 20px;
}
