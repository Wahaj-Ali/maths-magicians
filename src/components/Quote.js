import React from 'react';
import Header from './Header';

const Quote = () => {
  <Header />;
  return (
    <div>
      <p>
        Without mathematics, thereâ€™s nothing you can do. Everything around you is mathematics.
        Everything around you is numbers.
      </p>
      <big>
        <b>Shakuntala Devi, Indian writer and mental calculator</b>
      </big>
    </div>
  );
};

export default Quote;
