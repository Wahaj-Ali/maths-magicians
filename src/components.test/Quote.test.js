import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Quote from '../components/Quote';

describe('Check if it', () => {
  it('renders Quote component', () => {
    render(<Quote />);
  });
});

it('renders correctly when there are no items', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
