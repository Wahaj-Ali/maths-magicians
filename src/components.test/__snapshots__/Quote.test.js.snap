// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
<div
  className="quotePage"
>
  <p>
    Without mathematics, thereâ€™s nothing you can do. Everything around you is mathematics. Everything around you is numbers.
  </p>
  <big>
    <b>
      Shakuntala Devi, Indian writer and mental calculator
    </b>
  </big>
</div>
`;
